src/org/zaproxy/zap/extension/selenium/ExtensionSelenium.java

```diff
@@ -110,6 +113,9 @@
      * be closed when ZAP is closed.
      */
     private List<WebDriver> proxiedWebDrivers = new ArrayList<WebDriver>();
+    
+    private List<ProvidedBrowsersComboBoxModel> providedBrowserComboBoxModels =
```

Comment:
We should wrap the models in a `WeakReference` to not prevent them from being GC'ed.